addons/adminMenu/CfgFunctions.hpp

```diff
@@ -0,0 +1,12 @@
+class CfgFunctions {
+    class A3 {
+        tag = "BIS";
+        project = QUOTE(ADDON);
```

Comment:
Probably remove tag and project because those effect all of `A3` which I don't think we want to be messing with.